Description: 
 Patch is applied to set PREFIX, LIBDIR and DESTDIR by rules file.
 Also fixing ldconfig
Author: Jaromír Mikeš <mira.mikes@seznam.cz>
Forwarded: no

---
 libs/Makefile |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- zita-alsa-pcmi.orig/libs/Makefile
+++ zita-alsa-pcmi/libs/Makefile
@@ -20,9 +20,9 @@
 
 # Modify as required.
 #
-PREFIX = /usr/local
+PREFIX ?= /usr/local
 SUFFIX := $(shell uname -m | sed -e 's/^unknown/$//' -e 's/^i.86/$//' -e 's/^x86_64/$/64/')
-LIBDIR = lib$(SUFFIX)
+LIBDIR ?= lib$(SUFFIX)
 
 
 MAJVERS = 0
@@ -52,7 +52,7 @@ install:	$(ZITA-ALSA-PCMI_MIN)
 	install -Dm 644 $(ZITA-ALSA-PCMI_H)   $(DESTDIR)$(PREFIX)/include/$(ZITA-ALSA-PCMI_H)
 	install -Dm 644 $(ZITA-ALSA-PCMI_MIN) $(DESTDIR)$(PREFIX)/$(LIBDIR)/$(ZITA-ALSA-PCMI_MIN)
 	ln -sf $(ZITA-ALSA-PCMI_MIN) $(DESTDIR)$(PREFIX)/$(LIBDIR)/$(ZITA-ALSA-PCMI_SO)
-	ldconfig
+	/sbin/ldconfig -n $(DESTDIR)$(PREFIX)/$(LIBDIR)
 
 uninstall:
 	rm -rf $(DESTDIR)$(PREFIX)/include/zita-alsa-pcmi.h
